presto-spi/src/main/java/com/facebook/presto/spi/connector/ConnectorAccessControl.java

```diff
@@ -123,4 +123,11 @@
      * @throws com.facebook.presto.spi.security.AccessDeniedException if not allowed
      */
     void checkCanGrantTablePrivilege(Identity identity, Privilege privilege, SchemaTableName tableName);
+
+    /**
+     * Check if identity is allowed to revoke the specified privilege on the specified table from any user.
+     *
```

Comment:
Looks like all other method declarations follow this pattern of having a new line between the two comments. I am actually going to insert a new line in the comment for `checkCanGrantTablePrivilege` to make it consistent with other method declarations in this file.